main-project/src/members/repository/host-members.repository.ts

```diff
@@ -1,19 +1,15 @@
-import { EntityRepository, Repository } from 'typeorm';
+import { EntityRepository, InsertResult, Repository } from 'typeorm';
 import { HostMembers } from '../entity/host-members.entity';
 
 @EntityRepository(HostMembers)
 export class HostMembersRepository extends Repository<HostMembers> {
-  async addHostMembers(meetingNo, host) {
-    const hostMember = await this.createQueryBuilder('host_members')
+  async addHostMembers(hostsInfo): Promise<number> {
```

Comment:
```suggestion
  async saveHostMembers(hostsInfo): Promise<number> {
```
HostMember를 ```host_members``` 테이블에 저장하는 메서드가 맞지요?
그렇다면, add 보다는 save가 많이 사용돼요.

Repository에서 저장하는 메서드 명은 일반적으로 save가 사용되고 있어요.
Repository가 아닌 경우에는 set이 주로 사용됩니다.
```get``` ```set``` 메서드 많이 본 적 있죠?
```get```은 조회할 때, ```set``` 값을 세팅(저장)할 때 많이 사용해요.

add는 어느 멤버 변수에 새로운 값이 더해지는 개념으로 많이 사용되는 것 같아요.
데이터가 추가될 때도 사용되는 경우가 있긴 한데, 흔한 케이스는 아닌 것 같아요.
새로운 데이터를 추가할 때는 add 보다는 insert가 많이 이용됩니다.

add의 뜻 자체가 추가하다라는 의미도 있긴 하지만, 데이터가 추가될 때는 보통 insert를 사용하는 것 같아요.